apps/chrome-devtools/package.json

```diff
@@ -13,18 +13,17 @@
   ],
   "scripts": {
     "build": "yarn nx build chrome-devtools",
-    "postbuild:patch": "yarn patch:manifest && yarn patch:extension && yarn patch:package",
+    "postbuild:patch": "yarn patch:package && yarn patch:manifest && yarn patch:extension",
     "patch:package": "cpy 'package.json' 'dist' && patch-package-json-main",
     "copy:assets": "cpy 'src/manifest.json' 'src/devtools.html' dist --flat && cpy './src/assets/**' dist",
     "patch:manifest": "node scripts/sanitize-manifest.cjs",
     "build:extension": "tsc -b tsconfig.extension.json",
-    "patch:extension": "node scripts/sanitize-extension.cjs",
+    "patch:extension": "node scripts/sanitize-extension.cjs && node scripts/set-manifest-version.cjs",
     "start": "concurrently 'yarn:watch:*'",
     "watch:panel": "yarn build:panel --watch",
     "watch:components": "yarn build:components --watch",
     "watch:manifest": "yarn patch:manifest --watch",
     "watch:extension": "tsc-watch -b tsconfig.extension.json --noClear --onSuccess 'yarn patch:extension'",
-    "prepare:publish": "prepare-publish ./dist && node scripts/set-manifest-version.cjs",
```

Comment:
This is used by the `prepare-publish` task. You may need to keep `prepare-publish ./dist` or remove the task

Reply:
I removed the task it was not called in this package

Reply:
It's called at the workspace level for all the packages.
If I remember well, it is updating the package.json to add the common information (such as the contributors).
What I don't know it is if this added information is meaningful in the context of Chrome Extension of if this information should be set in the manifest.json instead.
But not sure it should be removed without be replaced by equivalent :S.